base_layer/core/src/base_node/consts.rs

```diff
@@ -23,6 +23,12 @@
 use std::time::Duration;
 
 /// The allocated waiting time for a request waiting for service responses from remote base nodes.
-pub const BASE_NODE_SERVICE_REQUEST_TIMEOUT: Duration = Duration::from_secs(180);
+pub const GENERAL_SERVICE_REQUEST_TIMEOUT: Duration = Duration::from_secs(180);
+/// The allocated waiting time for a fetch blocks request waiting for service responses from remote base nodes.
+pub const FETCH_BLOCKS_SERVICE_REQUEST_TIMEOUT: Duration = Duration::from_secs(30);
+/// The allocated waiting time for a fetch UTXOs request waiting for service responses from remote base nodes.
+pub const FETCH_UTXOS_SERVICE_REQUEST_TIMEOUT: Duration = Duration::from_secs(600);
+/// The minimum allocated waiting time for any request waiting for service responses from remote base nodes.
+pub const SERVICE_REQUEST_MINIMUM_TIMEOUT: Duration = Duration::from_secs(10);
 /// The fraction of responses that need to be received for a corresponding service request to be finalize.
 pub const BASE_NODE_SERVICE_DESIRED_RESPONSE_FRACTION: f32 = 0.6;
```

Comment:
I think this should also be removed and added to the config file with a default

Reply:
Constants removed in favour of moving the values directly into the default struct method, however, `desired_response_fraction` bit added to the config file, as we do not have a requirement for that to be user-configurable at the moment.